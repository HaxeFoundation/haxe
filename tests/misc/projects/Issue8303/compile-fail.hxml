-main Main
-D eval-call-stack-depth=20
--interp