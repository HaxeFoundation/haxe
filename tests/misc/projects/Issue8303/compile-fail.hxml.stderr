Uncaught exception Stack overflow
Main.hx:1: character 1 : Called from here
Main.hx:8: characters 4-9 : Called from here
Main.hx:8: characters 4-9 : Called from here
Main.hx:10: characters 3-8 : Called from here
Main.hx:3: characters 3-9 : Called from here
